src/main/java/sx/blah/discord/api/internal/DiscordClientImpl.java

```diff
@@ -211,30 +212,64 @@ public IUser getOurUser() {
 		return ourUser;
 	}
 
-	@Override
-	public List<IRegion> getRegions() {
-		if (REGIONS.isEmpty()) {
-			VoiceRegionObject[] regions = REQUESTS.GET.makeRequest(
-					DiscordEndpoints.VOICE+"regions", VoiceRegionObject[].class);
+	private synchronized void loadStandardRegions() {
+		if (regions.isEmpty()) { // Guarantee so standard regions are first
+			VoiceRegionObject[] regionObjects = RequestBuffer.request(() ->
+					(VoiceRegionObject[]) REQUESTS.GET.makeRequest(
+							DiscordEndpoints.VOICE + "regions", VoiceRegionObject[].class)).get();
+
+			Arrays.stream(regionObjects)
+					.map(DiscordUtils::getRegionFromJSON)
+					.forEach(r -> regions.putIfAbsent(r.getID(), r));
+		}
+	}
 
-			Arrays.stream(regions)
+	public synchronized IRegion getGuildRegion(Guild guild) {
+		loadStandardRegions();
```

Comment:
I don't think this is nescessary

Reply:
It is necessary since that method can be called from `IGuild#getRegion` before any of the other methods, which means it may not ever set all the previous standard regions beforehand.

Reply:
Fair enough I guess